collections/_release_4_5/entry-systems-input-sdl3-gamepad-input-driver.md

```diff
@@ -0,0 +1,27 @@
+---
+type: entry
+section: systems
+subsection: input
+rank: 0
+importance: 2
+anchor: sdl3-gamepad-input-driver
+title: SDL3 gamepad input driver
+blockquote: Gamepads galore!
+text: |
+  Gamepads are nowadays a given in modern PC gaming. Users expect their gamepad to just plug in and work. Not only that, but in order to deliver unique experiences, some gamepads are introducing new features.
```

Comment:
```suggestion
  Gamepads are a given in modern PC gaming. Users expect their gamepad to just plug in and work. Not only that, but in order to deliver unique experiences, some gamepads are introducing new features.
```
Could a few of these new features for unique experiences be listed? (Adaptive triggers, touchpads, etc?) This sentence piques my interest about them, but then doesn't follow up with details.

On the other hand, given that 4.5 doesn't actually directly support those new features, I could see it being misleading to bring them up

Reply:
Replaced with 
>   Gamepads are a given in modern PC gaming. Users expect their gamepad to just plug in and work. Not only that, but in order to deliver unique experiences, some gamepads are introducing new features; from adaptive triggers to advanced haptic feedback, as well as microphones and motion controls.